pyproject.toml

```diff
@@ -1,65 +1,57 @@
 [build-system]
-requires = [
-    "setuptools>=61.0.0",
-]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "nomad-analysis"
-version = '0.1.0'
-description = "A standard NOMAD plugin for analysis."
+version = '1.0'
+description = "A NOMAD plugin for analysis of FAIR data."
 readme = "README.md"
 authors = [
-    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" }
+    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" },
+    { name = "Jose M. Pizarro", email = "jose.pizarro@physik.hu-berlin.de" },
+    { name = "Sherjeel Shabih", email = "sherjeel.shabih@physik.hu-berlin.de" }
 ]
 license = { text = "Apache-2.0" }
 requires-python = ">=3.9"
-classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.9",
-    "License :: OSI Approved :: Apache Software License",
-]
```

Comment:
What is the reason for removing the classifiers?

Reply:
It surprises me we have to add these manually... and might make maintenance of the packages very cumbersome. I would assume that these are defined in other parts of the TOML, or better said, that pypi can grep these from somewhere.

I will read more next week.

Reply:
Ok, I understood that these are automatically assigned:

https://python-poetry.org/docs/pyproject/#classifiers

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers

However, I am not fully against defining them (and I can restore them back if you prefer so), thought I reiterate my concern: that maintaining these for ~60-70 plugins might be, in practice, very annoying.